### PR TITLE
povray 3.7.0.3

### PR DESCRIPTION
**Pull request by JCount**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by fxcoudert** [comment]
Looks like `patch-unix-configure.ac.diff`, `patch-unix-prebuild.sh.diff` and `boost-sharedptr.diff` should go away.

**Comment by JCount** [comment]
I am working on it locally, there have been some changes in the general unix support.

File: Formula/povray.rb [hunk 1]
```diff
@@ -16,46 +15,38 @@ class Povray < Formula
...
+    sha256 "0995431ffd56763fd821d3e65f53c2d799ddda0dca8859513e162e12f780cde0"
   end
 
-  # prebuild.sh doesn't create Makefile.in properly
-  patch :p0 do
-    url "https://raw.githubusercontent.com/Homebrew/formula-patches/3316200/povray/patch-unix-prebuild.sh.diff"
-    sha256 "1322acc324327019a31a2b97a59dbcfc6ca24981a833627399ae2ebd92c24ef6"
+  # Fixes configure script's stat usage
+  patch do
+    url "https://raw.githubusercontent.com/Homebrew/formula-patches/b444c92a9e1/povray/patch-unix-configure.ac.diff"
```
Comment (fxcoudert) [inline Formula/povray.rb]: Are you sure this is still needed? They changed to the current `stat --format=%u ${HOME} || stat -f %u ${HOME}` after I reported the issue. (It emits a warning to stderr, but works fine.)

File: Formula/povray.rb [hunk 2]
```diff
@@ -16,46 +15,38 @@ class Povray < Formula
   depends_on :macos => :lion
   depends_on "autoconf" => :build
   depends_on "automake" => :build
+  # Reported upstream 2017-09-02
+  # https://github.com/POV-Ray/povray/issues/317
+  depends_on "boost@1.60"
```
Comment (fxcoudert) [inline Formula/povray.rb]: Would you accept a patch to remove the wrong header includes, instead of depending on an old boost?

Comment (ilovezfs) [reply Formula/povray.rb]: if it gets submitted upstream :)

Comment (ilovezfs) [reply Formula/povray.rb]: Here's https://github.com/POV-Ray/povray/pull/318 adapted for stable:
https://github.com/Homebrew/formula-patches/pull/173

Comment (ilovezfs) [reply Formula/povray.rb]: @JCount note that it also means `ENV.cxx11` needs to be added.

File: Formula/povray.rb (hunk 1 above)
Comment (JCount) [reply Formula/povray.rb]: I figured I might as well just remove the warning as well, but I suppose one less patch is better.

**Pull request closed without merging**